pkg/workload/schemachange/schemachange.go

```diff
@@ -193,7 +193,7 @@ func (s *schemaChange) Ops(
 	// only for supported schema changes.
 	declarativeOpWeights := make([]int, len(opWeights))
 	for idx, weight := range opWeights {
-		if opDeclarative[idx] {
+		if opDeclarativeVersion[opType(idx)] > 0 {
```

Comment:
nit: no strong feeling just personal preference that to check if a key exists in a map, I use
```
if _, ok := m[key]; ok {
   ...
}
```
instead of checking if the return is the zero-value of the value type.

Reply:
Fixed! Not sure why I opted for the check this way 🤔